src/main/java/core/basesyntax/Machine.java

```diff
@@ -2,7 +2,7 @@
 
 public abstract class Machine {
 
-    String machineTitle;
+    protected String machineTitle;
```

Comment:
```suggestion
    private String machineTitle;
```
it is one of the OOP principal: encapsulation

Reply:
Done. And added getter for machineTitle.